src/com/yaji/Harness.java

```diff
@@ -0,0 +1,51 @@
+package com.yaji;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Harness {
+        private Date purchaseDate, lastDate;
+        private double  size;
+        private Integer gender;
```

Comment:
Use _primitives_ when you can. That means, use `int` instead of `Integer`, just like you use `double` and not `Double`.

Reply:
Hvorfor skal man bruge primitives i stedet? Gør det nogen forskel?

Reply:
Der er et par grunde:
1: Det er hurtigere, da du ikke skal lave et ny objekt (primitives bliver håndteret anderledes under the hood).
2: Det er god kodestil
3: Når man skal til at sammenligne tal er primitiver at foretrække. Hvis du bruger objekter kan du fx IKKE skrive "new Integer(1) < new Integer(2)", men derimod godt skrive "1 < 2". Desuden vil "new Integer(1) == new Integer(2)" give false, da det tester for memory-lokation, hvorimod "int a=1; int b=1; a==b" tester for værdien (igen, primitives bliver håndteret anderledes under the hood).